tests/lib/external/snapd-testing-tools/spread.yaml

```diff
@@ -53,14 +54,18 @@ backends:
             HTTPS_PROXY: 'http://squid.internal:3128'
             NO_PROXY: 'canonical.com'
         systems:
-            - ubuntu-20.04-64:
-                image: ubuntu-focal-20.04-amd64
             - ubuntu-22.04-64:
-                image: ubuntu-jammy-22.04-amd64
-            - centos-9-64:
-            - debian-12-64:
+                image: snapd-spread/ubuntu-22.04-64
+            - ubuntu-24.04-64:
+                image: snapd-spread/ubuntu-24.04-64
             - fedora-40-64:
+                image: snapd-spread/fedora-40-64
             - opensuse-15.5-64:
+                image: snapd-spread/opensuse-15.5-64
```

Comment:
How are those changes related?

Reply:
it is not but it was already changed in snapd-testing-tools